### ISSUE TITLE
More decimals

### ISSUE THREAD

**Issue by tkrunning** [mvarnagiris/financius#362]
Would be great to have the option of adding more than 2 decimals to certain currencies. In not able to add a transaction for 0.002 BTC for example...

**Comment by silin** [comment]
Agree. This issue should be blocker issue. It is very important. For example, in my country exchange rate for local currency is ~15 000 for 1 $. But the app use 0.0001 coefficient for exchange rate (mistake is about 50%!!!). As a result, the app shows invalid information about expenses (mistake of 50% for every item is very very significant issue). So especially for me this issue makes the app useless

**Comment by piatrouski** [comment]
Same issue!
